mobile/Screen/EventDetailScreen.js

```diff
@@ -1,19 +1,84 @@
 import React from 'react';
 import {
-  Title, Text, View,
-} from 'react-native-paper';
+  StyleSheet, View, Text, ScrollView, Image,
+} from 'react-native';
+import PropTypes from 'prop-types';
 
-function EventDetailScreen() {
+const styles = StyleSheet.create({
+  container: {
+    flex: 3,
+    backgroundColor: '#fff',
+    // alignItems: 'center',
+    // justifyContent: 'center',
+  },
+  square: {
+    width: 40,
+    height: 40,
+    backgroundColor: '#D9D9D9',
+    margin: 40,
+  },
+});
+
+function EventDetailScreen({
+  route,
+}) {
+  const {
+    title, date, day, location, time, summary,
+  } = route.params;
+  console.log(title);
   return (
-    <View>
-      <Title>
-        Event details
-      </Title>
+    <ScrollView style={styles.container}>
+      <Text style={{ fontSize: 30, fontWeight: 'bold' }}>{title}</Text>
+      <Text>{date}</Text>
+      <Text style={{ fontSize: 15, fontWeight: '10' }}>{location}</Text>
+      <View style={{
+        borderBottomColor: '#8A8A8A',
+        borderBottomWidth: '1.5',
+      }}
+      />
+      <View
+        style={[
+          styles.square,
+          { flexDirection: 'row' },
+          { flex: 3 },
+        ]}
+      />
+      <Text>{day}</Text>
+      <View style={styles.square} />
+      <Text>{time}</Text>
+      <View style={styles.square} />
+      <Text>{location}</Text>
+      <View style={{
+        borderBottomColor: '#8A8A8A',
+        borderBottomWidth: '1.5',
+      }}
+      />
+      <Text style={{ fontSize: 20, fontWeight: '30' }}>Event Description</Text>
```

Comment:
Not a big deal, but it is better to put styles into `StyleSheet` instead of inlining